WalletWasabi/FeeRateEstimation/FeeRateProviders.cs

```diff
@@ -85,11 +86,14 @@ private static async Task<FeeRateEstimations> GetFeeRateEstimationsAsync(string
 		using var response = await httpClient.GetAsync(apiEndPoint, cancellationToken).ConfigureAwait(false);
 		response.EnsureSuccessStatusCode($"Error requesting fee rate estimations to '{providerName}'");
 		var json = await response.Content.ReadAsStringAsync(cancellationToken).ConfigureAwait(false);
-		Logger.LogDebug(json);
 		return Result<FeeRateEstimations, Exception>
 			.Catch(() => extractor(json))
 			.Match(
-				estimations => estimations,
+				estimations =>
+				{
+					Logger.LogInfo($"Fetched fee rate from {providerName}: {estimations.GetFeeRate(confirmationTarget: 2)}.");
```

Comment:
```suggestion
					Logger.LogInfo($"Fetched fee rates from {providerName}: priority fee {estimations.GetFeeRate(confirmationTarget: 2)}.");
```

or something like this, is more clear imo

---

and could we change sat/B to sat/vB or s/vB?

`FeeRateProviders.GetFeeRateEstimationsAsync (94)	Fetched fee rate from Blockstream: 3.312 Sat/B.`